File: QuizApp-Android/QuizApp/app/src/main/java/edu/gatech/seclass/quizapp/QuizAppQuestion.java
package edu.gatech.seclass.quizapp;
import java.util.Date;

/**
 * Class describing the attributes of a question attempt that is used in the spaced repetition
 * algorithm. The question will always be a flashcard question.
 */

public class QuizAppQuestion {
    public String id;
    public String question;
    public String answer;
    public int chapter;
    public int section;

    public QuizAppQuestion(String id, String question, String answer,
                           int chapter, int section) {
        this.id = id;
        this.question = question;
        this.answer = answer;
        this.chapter = chapter;
        this.section = section;
    }
}

File: QuizApp-Android/backend/QuizAppQuestion.js
const mongoose = require('mongoose');

const QuizAppQuestionSchema = new mongoose.Schema({
	questionId: String,
	question: String,
	answer: String,
	chapter: Number,
	section: Number,
});
module.exports = mongoose.model(
	'QuizAppQuestion',
	QuizAppQuestionSchema,
	'QuizAppQuestion'
);
